src/middleware/rateLimiter.js

```diff
@@ -1,24 +1,56 @@
 const rateLimit = require("express-rate-limit");
+const { MemoryStore } = require("express-rate-limit");
+ 
 
+const limiter = rateLimit({
+  windowMs: 1000,
+  max: 3,
+  //message: "Your limit exceeded",
+  standardHeaders: true,
+  legacyHeaders: false,
+  handler: function (req, res) {
+    return res.status(429).json({
+      error: "You sent too many requests. Please try again later",
+    });
+    },
+  	store: new MemoryStore(),
```

Comment:
Codacy has a fix for the issue: [Delete `↹`](https://app.codacy.com/gh/armerillo/ql-task/pullRequest?prid=9976636)
```suggestion
  store: new MemoryStore(),
```